x-pack/plugins/reporting/public/components/screen_capture_panel_content.tsx

```diff
@@ -85,22 +89,13 @@ export class ScreenCapturePanelContent extends Component<Props, State> {
     const el = document.querySelector('[data-shared-items-container]');
     const bounds = el ? el.getBoundingClientRect() : { height: 768, width: 1024 };
 
-    if (this.props.reportType === 'png') {
-      return {
-        dimensions: {
-          height: bounds.height,
-          width: bounds.width,
-        },
-      };
-    } else {
-      return {
-        id: 'preserve_layout',
-        dimensions: {
-          height: bounds.height,
-          width: bounds.width,
-        },
-      };
-    }
+    return {
+      id: this.props.reportType === 'png' ? 'png' : 'preserve_layout',
```

Comment:
NOTE: the previous code would have an undefined `id` field when the export type was PNG. This changes it to have an `id: 'png'` value. There doesn't seem to be any functional change from this

Reply:
Interesting, good catch. At least it's less ambiguous now.